source/expressions.tex

```diff
@@ -4683,10 +4686,13 @@
 the coefficient $\tcode{r}_i$
 of the base-2 representation of the result \tcode{r}
 is 1 if $\tcode{x}_i$ is 0, and 0 otherwise.
+\begin{note}
+The result is the ones' complement of the operand (both considered as unsigned).
```

Comment:
I find "both" to be referring to "operand" and "ones' complement", which seems unintended.

Maybe just drop "both" and leave it at that, considering this is a note anyway?

Reply:
Done, PTAL.

Reply:
I did actually mean both of those things: for a signed operand, you pretend that it's unsigned (with the same bits) and take whichever kind of complement, producing another unsigned value, but that's the result only when pretending that _it_ is unsigned in the same fashion.  That said, if people think it's clear without the "both", that works for me.

Reply:
How about "(result and operand considered as unsigned)"?

Reply:
Works for me.

Reply:
Done.